AWS X-Ray integration documentation and cleanup (#8601)
DataDog/documentation: content/en/serverless/installation/ruby.md

After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][8].

> ### Monitor Custom Serverless Metrics

Author: ```suggestion
### Monitor Custom Business Metrics
```

    end
end
```

> Learn more about submitting custom metrics [here][9].

Author: ```suggestion
[Enable custom metric submission][3] to get started.
```

### Enable the AWS X-Ray Integration

> Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][10].

Reviewer: same feedback about `trace data`

> Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][10].

Author: ```suggestion
Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, [choose between setting up Datadog APM or the AWS X-Ray integration][5] for your tracing needs.
```
